src/app/account/facility/facility.component.html

```diff
@@ -5,61 +5,94 @@ <h3>Facility Information</h3>
-                    </label>
-                    <br>
-                    <label>Teir
-                        <input type="text" formControlName="tier" (input)="onFormChange();">
-                    </label>
-                    <br>
-                    <label>Size
-                        <input type="text" formControlName="size" (input)="onFormChange();">
-                    </label>
-                    <br>
-                    <label>Units
-                        <input type="text" formControlName="units" (input)="onFormChange();">
-                    </label>
-                    <br>
-                    <label>Division
-                        <input type="text" formControlName="division" (input)="onFormChange();">
-                    </label>
-                </form>
+    <h4>Facilty Information</h4>
+    <br>
+    <div class="row">
+        <div class="col-lg-4 profile-image">
+            <img src="https://placehold.it/350x200" alt="">
+            <form>
+                <label class="profile-image-input" for="img"><i class="fa fa-upload"></i> Upload Image
+                    <input type="file" id="img" name="img" accept="image/*">
+                </label>
+            </form>
+        </div>
+        <div class="col-lg-8">
+            <br>
+            <div class="row">
+                <div class="col-lg-6">
+                    <form [formGroup]="facilityForm" autocomplete="on">
+                        <label>Building / Area
+                            <input type="text" formControlName="name" (input)="onFormChange();">
+                        </label>
+                        <br>
+                        <label>County
+                            <input type="text" formControlName="country" (input)="onFormChange();">
+                        </label>
+                        <br>
+                        <label>State
+                            <input type="text" formControlName="state" (input)="onFormChange();">
+                        </label>
+                        <br>
+                        <label>Address
+                            <input type="address" formControlName="address" (input)="onFormChange();" autocomplete="shipping street-address">
+                        </label>
+                    </form>
+                </div>
+                <div class="col-lg-6">
+                    <form [formGroup]="facilityForm">
+                        <label>Business Type
+                            <input type="text" formControlName="type" (input)="onFormChange();">
+                        </label>
+                        <br>
+                        <label>Teir
+                            <input type="text" formControlName="tier" (input)="onFormChange();">
+                        </label>
+                        <br>
+                        <label>Size
+                            <input type="text" formControlName="size" (input)="onFormChange();">
```

Comment:
Should size be a number or text?

Reply:
These inputs will possibly change. Waiting on Paulomi to give me the exact fields before editing these inputs.